### PR TITLE
[SSL] Impacts of LE chain update on different certificate types

### PR DESCRIPTION
**Pull request by RebeccaTamachiro**
PCX-10503

### CODE REVIEW

File: content/ssl/reference/migration-guides/lets-encrypt-chain.md [hunk 1]
```diff
@@ -10,58 +10,77 @@ meta:
 
 Let's Encrypt - one of the [certificate authorities (CAs)](/ssl/reference/certificate-authorities/) used by Cloudflare - has announced changes in its [chain of trust](/ssl/concepts/#chain-of-trust).
 
-As the IdenTrust cross-sign (DST Root CA X3) expires on **September 30, 2024**, the self-signed ISRG Root X1 will be the only chain used for RSA certificates issued through Let's Encrypt.
+As the IdenTrust cross-sign (DST Root CA X3) expires on **September 30, 2024**, the self-signed ISRG Root X1 will be the only chain used for RSA certificates issued through Let's Encrypt. Refer to [background](#background) or our [blog](https://blog.cloudflare.com/shortening-lets-encrypt-change-of-trust-no-impact-to-cloudflare-customers) for details.
```
Comment (Maddy-Cloudflare) [inline content/ssl/reference/migration-guides/lets-encrypt-chain.md]:
```suggestion
As the IdenTrust cross-sign (DST Root CA X3) expires on **September 30, 2024**, the self-signed ISRG Root X1 will be the only chain used for RSA certificates issued through Let's Encrypt. Refer to [Background](#background) or our [blog](https://blog.cloudflare.com/shortening-lets-encrypt-change-of-trust-no-impact-to-cloudflare-customers) for details.
```

File: content/ssl/reference/migration-guides/lets-encrypt-chain.md [hunk 2]
```diff
@@ -10,58 +10,77 @@ meta:
...
 
-This means that, if visitors to your website or application are using older devices (e.g. Android 7.0 and earlier), or if their systems rely solely on the cross-signed chain, they may see warning messages or fail to connect to your services.
+To minimize impact, besides communicating the changes and providing recommendations early, Cloudflare will proceed as follows:
+
+* Automatically switch to a different certificate authority all Cloudflare-managed certificates where you have not specifically chosen Let's Encrypt.
```
Comment (Maddy-Cloudflare) [inline content/ssl/reference/migration-guides/lets-encrypt-chain.md:17]: Does the order matter here? If it does, it should be a numbered list. :)

File: content/ssl/reference/migration-guides/lets-encrypt-chain.md [hunk 3]
```diff
@@ -10,58 +10,77 @@ meta:
...
+* Notify you of Cloudflare-managed certificates that you specifically set to use Let's Encrypt, and whether hostnames covered by these certificates are receiving requests from the [impacted devices or systems](#impact).
+* Bundle Let’s Encrypt certificates uploaded to Cloudflare with the appropriate chains, as long as the [bundling method](/ssl/edge-certificates/custom-certificates/bundling-methodologies/) is set to compatible or modern.
+
+Refer to the sections below for more information on each of these actions and how they may reflect on the different certificate types.
+
+## Client impact
+
+The expiration of the cross-signed chain will primarily affect older devices, for example Android 7.0 and earlier. And systems that solely rely on the cross-signed chain, lacking the ISRG Root X1 chain in their [trust store](/ssl/concepts/#trust-store), will also be affected.
```
Comment (Maddy-Cloudflare) [inline content/ssl/reference/migration-guides/lets-encrypt-chain.md]:
```suggestion
The expiration of the cross-signed chain will primarily affect older devices, for example Android 7.0 and earlier. Systems that solely rely on the cross-signed chain, lacking the ISRG Root X1 chain in their [trust store](/ssl/concepts/#trust-store), will also be affected.
```

**Review by Maddy-Cloudflare — APPROVED** [review]

File: content/ssl/reference/migration-guides/lets-encrypt-chain.md (hunk 2 above)
Comment (RebeccaTamachiro) [reply content/ssl/reference/migration-guides/lets-encrypt-chain.md:17]: Thanks! No, in this case, those are independent scenarios/actions.

**Comment by RebeccaTamachiro** [comment]
Moved to https://github.com/cloudflare/cloudflare-docs/pull/14050

**Pull request closed without merging**